backend/apps/slack/admin/bot_interaction.py

```
@@ -0,0 +1,80 @@
+        "nest_updated_at",
+    )
+    search_fields = (
+        "channel_id",
+        "user_id",
+        "user_message",
+        "intent_category",
+    )
+ 
+    @admin.display(description="Message")
+    def short_message(self, obj):
+        """Truncated user message for list display."""
```

Ruff (0.15.18)

[warning] First line of docstring should be in imperative mood: "Truncated user message for list display."

(D401)